### DISCUSSION TITLE
Another supported model

### DISCUSSION THREAD

**Discussion by ncrutch13** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#167] [Show and tell]
Not sure if this is the right place, but just as an FYI. 

I was able to build and install this and it is working fine. My model spa was not on the supported model list, so just wanted to put that on here in case it will help others.

Coleman (Bestway) SaluSpa Cali, Model #90437E
![IMG_1719](https://user-images.githubusercontent.com/4740699/147867196-8ab195c8-8488-4803-9178-d7f022ca9356.jpg)
![IMG_1752](https://user-images.githubusercontent.com/4740699/147867220-50a9d650-08e6-4b43-b7ff-247d0bc2dd02.jpg)

**Comment by visualapproach** [comment]
Thank you, it's in the list of models now.
